### PR TITLE
add local storage

### PR DESCRIPTION
**Pull request by belal19979**
add local storage

### CODE REVIEW

File: src/views/createNewScore.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import { CURRENT_SCORE_ID } from '../constants.js';
+import { quizData } from '../data.js';
+
+export const createScoreElement = () => {
```
Comment (arkaitzgarro) [inline src/views/createNewScore.js:6]: I don't get why you need to create an extra function, we already have it defined in `scoreView.js` file right?

**Review by arkaitzgarro — COMMENTED** [review]
It's going into the right direction, some small adjustments and we are done.

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -48,21 +53,25 @@ export const initQuestionPage = () => {
...
     }
 
     updateScore();
-
   };
 
-  answersListElement.addEventListener('click', correctAnswer);
+  answersListElement.addEventListener('click', (e) => {
+    correctAnswer(e);
+    localStorage.setItem('selectedAnswer', e.target.dataset.key);
```
[the remark this answers is not in the document]
Comment (belal19979) [reply src/pages/questionPage.js]: yes I made it now , but I have no idea where are we use the all answers from local storage so I did not call it any where but its in local storage

File: src/pages/questionPage.js [hunk 3]
```diff
@@ -19,6 +21,9 @@ export const initQuestionPage = () => {
 
   userInterface.appendChild(createScoreElement(quizData.finalScore));
 
+  let storedIndex = localStorage.getItem('currentQuestionIndex');
+  quizData.currentQuestionIndex = +storedIndex;
```
[the remark this answers is not in the document]
Comment (belal19979) [reply src/pages/questionPage.js]: yes thank you for this feed back

File: src/views/createNewScore.js (hunk 1 above)
Comment (belal19979) [reply src/views/createNewScore.js]: sorry by mistake

**Comment by arkaitzgarro** [comment]
@belal19979 I moved things around a bit, take a look to the code, and we can talk about it tomorrow.

**Review by arkaitzgarro — APPROVED** [review]

**Pull request merged**